installer/x86_64/install.sh

```diff
@@ -666,6 +666,11 @@ else # install_env = "onie"
     fi
 fi
 
+# Add extra linux command line
+extra_cmdline_linux=%%EXTRA_CMDLINE_LINUX%%
+echo "EXTRA_CMDLINE_LINUX=$extra_cmdline_linux"
+ONIE_PLATFORM_EXTRA_CMDLINE_LINUX="$ONIE_PLATFORM_EXTRA_CMDLINE_LINUX $extra_cmdline_linux"
```

Comment:
>ONIE_PLATFORM_EXTRA_CMDLINE_LINUX [](http://example.com/codeflow?start=0&length=33)

Do not combine %%EXTRA_CMDLINE_LINUX%% with ONIE_PLATFORM_EXTRA_CMDLINE_LINUX. Just add %%EXTRA_CMDLINE_LINUX%% to below menuentry template.

Reply:
Sorry, by semantics, GRUB_CMDLINE_LINUX should include %%EXTRA_CMDLINE_LINUX%%

Reply:
Thanks, fixed, better to use GRUB_CMDLINE_LINUX .